docs/reference/env-vars/tailpipe_temp_dir_max_mb.md

```diff
@@ -0,0 +1,17 @@
+---
+title: TAILPIPE_TEMP_DIR_MAX_MB
+sidebar_label: TAILPIPE_TEMP_DIR_MAX_MB
+---
+
+# TAILPIPE_TEMP_DIR_MAX_MB
+
+Define the maximum amount of disk space (in MB) that `tailpipe` is allowed to use for temporary storage of intermediate data, such as JSONL-formatted files used during processing.
+
+This limit applies to the `.tailpipe/internal` directory and ensures that large result sets or slow-consuming data pipelines do not exhaust available disk space.
+
```

Comment:
Can you disable the limit by setting it to `0` ?

Reply:
Yes. Updating